#! /usr/bin/env python
# Task Long configuration

PACKAGE='floor_nav'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *
from task_manager_lib.parameter_generator import *

gen = TaskParameterGenerator()
#       Name                    Type            Description               Default    Min   Max
gen.add("roi_x",         double_t, 0,    "X coordinate of the ROI",  0.)
gen.add("roi_y",         double_t, 0,    "Y coordinate of the ROI",  0.)
gen.add("roi_radius",    double_t, 0,    "size of the ROI",  1.0)

exit(gen.generate(PACKAGE, "task_manager_floor_nav", "TaskWaitForROI"))

